### PR TITLE
docs(route-handlers): add note about caching with other support HTTP methods in Route Handlers

### PR DESCRIPTION
**Pull request by samcx**
## Why?

We currently do not clarify that other supported HTTP Methods (all other besides `GET`) in Route Handlers are not cached. This remains true even if they are placed alongside a `GET` that is cached.

### CODE REVIEW

**Comment by samcx** [comment]
* **#70049** <a href="https://app.graphite.dev/github/pr/vercel/next.js/70049?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @samcx and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/02-app/01-building-your-application/01-routing/13-route-handlers.mdx [hunk 1]
```diff
@@ -46,7 +46,7 @@ In addition to supporting the native [Request](https://developer.mozilla.org/doc
 
 ### Caching
 
-Route Handlers are not cached by default. You can, however, opt into caching for `GET` methods. To do so, use a [route config option](/docs/app/api-reference/file-conventions/route-segment-config#dynamic) such as `export const dynamic = 'force-static'` in your Route Handler file.
+Route Handlers are not cached by default. You can, however, opt into caching for `GET` methods. Other support HTTP methods are **not** cached. To cache a `GET` method, use a [route config option](/docs/app/api-reference/file-conventions/route-segment-config#dynamic) such as `export const dynamic = 'force-static'` in your Route Handler file.
```
Comment (ijjk) [inline docs/02-app/01-building-your-application/01-routing/13-route-handlers.mdx:49]:
```suggestion
Route Handlers are not cached by default. You can, however, opt into caching for `GET` methods. Other HTTP methods are **not** cached. To cache a `GET` method, use a [route config option](/docs/app/api-reference/file-conventions/route-segment-config#dynamic) such as `export const dynamic = 'force-static'` in your Route Handler file.
```

Comment (samcx) [reply docs/02-app/01-building-your-application/01-routing/13-route-handlers.mdx:49]: @ijjk oops that should be supported*

**Review by ijjk — APPROVED** [review]

**Comment by samcx** [comment]
### Merge activity

* **Sep 12, 6:20 PM EDT**: @samcx merged this pull request with [Graphite](https://app.graphite.dev/github/pr/vercel/next.js/70049).

**Pull request merged**